src/main.rs

```diff
@@ -181,6 +198,24 @@ fn get_user_id_by_name(name: &str) -> Result<UserId, Box<Error>> {
     Ok(user.id)
 }
 
+fn list_issues(
+    config: Config,
+    project: &str,
+    filter_state: &MyIssueState,)
+    -> Result<String, Box<Error>> {
+    let gitlab_client = Gitlab::new(config.gitlab_domain, config.gitlab_token)?;
+    let project = gitlab_client.project_by_name(project)?;
+
+    gitlab_client.issues(project.id)
+        .and_then(|issues| {
+            issues.into_iter()
+            .filter(|i| i.state == filter_state.0)
+            .for_each(|i| println!("#{} {:?} {} {}", i.id, i.state, i.title, i.created_at.format("%F %H:%M")));
```

Comment:
Not sure about how it's usually done in Rust, but relying on `Debug` for regular display seems a bit odd.
Poke @Geal

Reply:
I can implement the `Trait Display` for `MyIssueState` to correct this if you want ?

Reply:
yes, `Debug` is not meant for the user side